Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.7
```

La versión actual es 3.8. Tal vez deberíamos dejar libre el 3 y solo anclarlo cuando tengamos un problema?

```suggestion
FROM python:3
```

Hoy, la version 3 apunta a 3.8.5